blog: FSWeekend 2024 NOTAM (#517)
flybywiresim/website: src/posts/fsweekend-2024.md

---
title: 'FlyByWire Simulations at FS Weekend 2024'
date: '2024-04-07'
authors:
  - 'straks'
category: 'INFO'
metaImage: '/img/notam-images/fsweekend2024/header.png'
metaAlt: 'FlyByWire Simulations at FS Weekend 2024'
---

A few weeks ago, over the weekend of March 16th and March 17th, 2024, the [FS Weekend](https://fsweekend.nl/) conference took place in the [Aviodrome Museum](https://www.aviodrome.nl/) in Lelystad in the Netherlands. This conference brings together Flight Simulation enthusiasts and professionals from around the world. The Aviodrome Museum presents the perfect environment for a flight simulation event with its rich presentation of the history of aviation through several exhibits and a large range of aircraft from different time periods.

FlyByWire Simulations was present at FS Weekend with a large booth and fourteen of our most dedicated developers and team members. We welcomed hundreds of fans to our booth who got to try out flying the A32NX and our new project, the A380X.

## Our Booth and Team

> For the FlyByWire Simulations team, events like this are the perfect time to meet up with each other in person and build our personal bonds even more, while also showing what an Free and Open Source developer group can achieve when working together on a passion in the flight simulation community.

Reviewer: ```suggestion
For the FlyByWire Simulations team, events like this are the perfect time to meet up with each other in person and build our personal bonds even more, while also showing what a Free and Open Source developer group can achieve when working together on a passion in the flight simulation community.
```

> As a team, we worked on designing a booth with great marketing material and graphics, making sure everything is in our branding style. We brought together skills and people from around the world to set up a booth that looked amazing and inviting to our visitors.

Reviewer: If possible coudl you expand upon the marketing materials here? It would be good to state like our digital displays, brochures, banners, etc. Since a lot of people worked on stuff to get printed and itemizing it would give a nice little "credit" on what was brought over.

[![FlyByWire Simulations Booth](/img/notam-images/fsweekend2024/booth.png)](/img/notam-images/fsweekend2024/booth.png)

The booth also had two setups where visitors could fly the A32NX and the A380X. A lot of people were very excited about flying the A380X as it was the first time we gave people the opportunity to actually fly it in Microsoft Flight Simulator 2020.

> Our booth was a huge success and there was always a buzz of activity with many fans coming up, sharing their experiences and asking questions about the organization, the different aircraft and our progress on different features. We also gave out a lot of stickers to anyone who visited, so hopefully you'll see these pop up in the wild more and more!

Reviewer: ```suggestion
Our booth was a huge success and there was always a buzz of activity with many fans coming up. They shared their experiences and asked questions about the organization, the different aircraft and our progress on different features. We also gave out a lot of stickers to anyone who visited, so hopefully you'll see these pop up in the wild more and more!
```

Just a readability suggestion.

## A32NX Live VATSIM Flight

For the second time in a row, our team set up a live flight with the A32NX on VATSIM. Both to promote our beautiful A32NX project, as well as promote VATSIM as an online ATC experience that enhances the realism of flight simulation.

The live flight took us from Groningen airport (EHGG) to Rotterdam (EHRD) in the Netherlands and we worked with the [Dutch division of VATSIM](https://www.dutchvacc.nl/) to have full coverage of our flight.

While two team members of FlyByWire Simulations were flying and communicating with ATC, one person from VATSIM and another of our crew members gave live commentary on the flight. This commentary explained how VATSIM worked and how to get started as a new pilot, or as a new controller.

[![A32NX Live VATSIM Flight Commentary](/img/notam-images/fsweekend2024/live-flight-commentary.png)](/img/notam-images/fsweekend2024/live-flight-commentary.png)

## What's Next?

> As FS Weekend 2024, our team started packing up and immediately started to think about the next event where we want to show you all the amazing things we do.

Reviewer: ```suggestion
As FS Weekend 2024 came to a close, our team started packing up and immediately started to think about the next event where we want to show you all the amazing things we do.
```

Seems like you trailed off thought here, added a small contextual note.

Reviewer 2: And FS Weekend -> FSWeekend
